governance/executive-council.md

```diff
@@ -0,0 +1,261 @@
+(pyos-executive-council)=
```

Comment:
Add a section - that says voted on and has a date... between operations and governance part. say revised on....

Reply:
done!